### ISSUE TITLE
dev testing for pendulum lab, version 1.0.0-dev.11

### ISSUE THREAD

**Issue by veillette** [phetsims/tasks#632]
We have resolved many issues on the pendulum lab.  We have deployed a new dev that corrects the length and mass range from dev,10
We would like to get some attention from the QA team. 

Link to dev version:
http://www.colorado.edu/physics/phet/dev/html/pendulum-lab/1.0.0-dev.11/pendulum-lab_en.html

Dev Test Platforms:

- [x] iPad 2 iOS 9.3.1]
- [x] Win Chrome

Please close any issues that have been correctly addressed. 
Please create separate issue for any problems that you find.

Assigning to @ariel-phet

**Comment by ariel-phet** [comment]
@alynne-phet @brroberts1231 @phet-steele please make this the next testing priority after #630

**Comment by ariel-phet** [comment]
Bumping to top priority

**Comment by phet-steele** [comment]
@veillette this testing is done, but let's talk on Skype for some layout fixes if you are up for it (or I could attempt to write up an issue).

**Comment by veillette** [comment]
Thanks @phet-steele for the work,  
For your information, we have been breaking a few things on master while we start using a modelviewtransform.
It would be probably easier to skype.
